.github/workflows/health-check.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '10.x', '12.x', '14.x' ]
+        node: [ '18.x', '20.x' ]
```

Comment:
i see PRs where we support 16 and 18 and others, like this one, that support 18 and 20, what are the actual versions we support?

Reply:
we support LTS versions